scripts/db_migrator.py

```diff
@@ -1200,6 +1200,55 @@ def set_version(self, version=None):
         entry = { self.TABLE_FIELD : version }
         self.configDB.set_entry(self.TABLE_NAME, self.TABLE_KEY, entry)
 
+    def enable_per_command_aaa(self):
+        aaa_table = self.configDB.get_table('AAA')
+        if not aaa_table:
+            log.log_info('AAA table does not exist ignore setup per-command accounting&authorization.')
+            return
+
+        authentication_config_old = self.configDB.get_entry('AAA', 'authentication')
+        # If device enabled TACACS authentication, then enable TACACS per-command accounting and authorization
```

Comment:
>accounting [](http://example.com/codeflow?start=82&length=10)

Is it better to use separate PRs for accounting and authorization enablement?

Reply:
Fixed, this PR only contains migrate accounting part. will submit another PR later.